app/assets/javascripts/backbone/views/breakeven_view.coffee

```diff
@@ -6,17 +6,27 @@ class MS.Views.BreakevenView extends Backbone.View
     'change .editable-table[data-table="fixed_costs"] input.amount': 'totalFixedCostsChanged'
 
   initialize: (e) ->
-    @$('.editable-breakeven-table').each (index, table) =>
-      new MS.Views.EditableBreakevenTableView(el: table)
+    tables = @$('.editable-table').map (index, table) =>
```

Comment:
Refactored `EditableTable` to cover what is needed for the `EditableBreakevenTable`